client/lib/feed-stream-store/feed-stream.js

```diff
@@ -441,7 +441,7 @@ assign( FeedStream.prototype, {
 			postKeys = this.filterNewPosts( data.posts );
 			if ( postKeys.length > 0 ) {
 				this.pendingPostKeys = postKeys;
-				this.pendingDateAfter = moment( data.date_range.after );
+				this.pendingDateAfter = moment( FeedPostStore.get( postKeys[ postKeys.length - 1 ] ).date );
```

Comment:
Can `FeedPostStore.get( postKeys[ postKeys.length - 1 ]` be null in some cases? The tests are complaining `TypeError: Cannot read property 'date' of undefined`: https://circleci.com/gh/Automattic/wp-calypso/3900